spidermon/contrib/scrapy/monitors.py

```diff
@@ -87,24 +88,48 @@ def test_should_finish_with_expected_reason(self):
 class UnwantedHTTPCodesMonitor(BaseScrapyMonitor):
     """Check for maximum number of unwanted HTTP codes.
 
+    Usage # 1
+
     You can configure a ``dict`` of unwanted HTTP codes with
     ``SPIDERMON_UNWANTED_HTTP_CODES`` the default value is::
 
-        DEFAULT_ERROR_CODES = {
+        SPIDERMON_UNWANTED_HTTP_CODES = {
             code: 10
-            for code in [400, 407, 429, 500, 502, 503, 504, 523, 540, 541]}
+            for code in [400, 407, 429, 500, 502, 503, 504, 523, 540, 541]
+        }
+
+    Usage # 2
+
+    You can configure an ``int`` of maximum unwanted HTTP codes with
+    ``SPIDERMON_UNWANTED_HTTP_CODES_MAX_COUNT`` the default value is::
+
+        SPIDERMON_UNWANTED_HTTP_CODES_MAX_COUNT = 10
+
+        ----->
+
+        SPIDERMON_UNWANTED_HTTP_CODES = {
+            code: SPIDERMON_UNWANTED_HTTP_CODES_MAX_COUNT
+            for code in [400, 407, 429, 500, 502, 503, 504, 523, 540, 541]
+        }
+
     """
+    DEFAULT_ERROR_MAX_COUNT = 10
+    ERROR_CODES = [400, 407, 429, 500, 502, 503, 504, 523, 540, 541]
 
-    DEFAULT_ERROR_CODES = {
-        code: 10 for code in [400, 407, 429, 500, 502, 503, 504, 523, 540, 541]
-    }
+    DEFAULT_ERROR_CODES = {code: 10 for code in ERROR_CODES}
 
     @monitors.name("Should not hit the limit of unwanted http status")
     def test_check_unwanted_http_codes(self):
-        error_codes = self.crawler.settings.getdict(
+        error_codes_dict = self.crawler.settings.getdict(
```

Comment:
Can't we read a list as well?
My idea is that, I can configure an arbitrary dict.
Or I can use the default _error count with some custom HTTP codes_, or use a _custom error count with default HTTP codes_.

Reply:
Yes I thought about that too.
But I think that would be kind of messy.
If the user wants to change the default error codes.
He can do that through dictionary setting.